oauth-server/cmd/service/main.go

```diff
@@ -0,0 +1,22 @@
+package main
```

Comment:
Why did you introduced this as a new service on the root plgd cloud project structure (folder)? It should be part of the authorization server. It's misleading to have it there and it's just a mock for testing and bundle.

Reply:
moved to test/oauth-server